helm/values.yaml

```diff
@@ -46,35 +46,6 @@ image:
   repository: exporter-trigger
   tag: 'latest'
 
-opa:
```

Comment:
do you need "jwtPublicKeySecretName: map-colonies-jwks" ?

Reply:
done